shuttle/db/db_functions.py

```diff
@@ -111,4 +113,64 @@ def portal_common_profile(username):
         if conn:
             conn.close()
         # if mybethel can't get the data, then prevent anything from loading
-        return abort(503)
\ No newline at end of file
+        return abort(503)
+
+
+def commit_schedule_to_db(table):
+    sql = "DELETE FROM SHUTTLE_SCHEDULE"
+    query(sql, 'write')
+
+    for i in range(len(table)):
+        location = ""
+        for j in range(len(table[i])):
+            try:
+                arrival_time = 0
+                if table[i][j].lower() == 'clc' or table[i][j].lower() == 'anderson' or table[i][j].lower() == 'pine tree' \
+                        or table[i][j].lower() == 'kresge' or table[i][j].lower() == 'scandia' or table[i][j].lower() == 'north':
```

Comment:
It would be better to figure out what column the locations are in and then just use that column number to get the values instead of hardcoding them in just in case they ever change.